pastis/temporal_analysis/harris_mode.ipynb

```diff
@@ -0,0 +1,3338 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "30a87e9f",
+   "metadata": {},
+   "source": [
+    "## This notebook contains all the scripts used to do segment level thermal sensitivity analysis and produce plots all plots required for presentation, report or article. The code block which are commented aren't necessary to do full e2e analysis, but are used to support report/article."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cb4388c",
+   "metadata": {},
+   "source": [
+    "###  Import neccessary python libraries and pastis functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c90040ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir(\"/Users/asahoo/repos/PASTIS\")\n",
```

Comment:
Please delete this, it should not be needed. If you find it is needed, then you need to fix the imports.

Reply:
still can't fix this but deleting the line for safe PR merge

Reply:
What is it that you cannot fix, maybe I can help?
Since pastis is a package from which we do all the imports, it should not be necessary for you to put yourself in the directory of the repo. What is it that is failing for you?